guides/common/modules/proc_configuring-a-host-to-use-the-pull-client.adoc

```diff
@@ -16,7 +16,7 @@ ifndef::foreman-el[]
 * The AppStream repository for the operating system version of the host is synchronized on {ProjectServer}, available in the content view and the lifecycle environment of the host, and enabled for the host.
 For more information, see {ContentManagementDocURL}Changing_the_Repository_Sets_Status_for-a-Host-in_{project-context}_content-management[Changing the repository sets status for a host in {Project}] in _{ContentManagementDocTitle}_.
 endif::[]
-* The host can communicate with its {SmartProxy} over MQTT using port `1883`.
+* The host can communicate with its {SmartProxy} over MQTT using port`1883` and `9090`.
```

Comment:
If I remember the ticket report correctly, it actually requests adding port 9090 as a requirement for the HTTPS service, which would mean line 20. This would require changes to your proposal for updating the tables in the port requirements files below -- those would also need line entries for HTTPS rather than MQTT.

Reply:
Oh yes! Corrected the protocol in all files.